nni/__main__.py

```diff
@@ -65,6 +66,7 @@ def main():
         tuner._on_error()
         if assessor is not None:
             assessor._on_error()
+        dispatcher.report_error(traceback.format_exc())
```

Comment:
If web socket already loses connection, will it end up here? What will `report_error` behave?